src/components/ApplyForm.js

```diff
@@ -41,7 +42,7 @@ function ApplyForm() {
 
   if (hasSent) {
     return (
-      <Card style={{ maxWidth: MAX_WIDTH, margin: 'auto' }}>
+      <Card className="uil-m-auto" style={{ maxWidth: MAX_WIDTH }}>
```

Comment:
Do you plan to get rid of this `style`?

Reply:
As much as possible, it is unfortunately not possible due to the lack of atomic classes defined from fragment.css documentation.

Reply:
We should be able to use class names with solutions like Styled Components. See [their `css` prop](https://www.styled-components.com/docs/api#css-prop).